.github/CONTRIBUTING.md

```diff
@@ -1,36 +1,169 @@
-# Contributing to BeEF
+# Contributing
+### Anyone is welcome to make BeEF better!
+Thank you for wanting to contribute to BeEF. It's effort like yours that helps make BeEF such a great tool.
 
-Anyone is welcome to make BeEF better!
+Following these guidelines shows that you respect the time of the developers developing this open source project and helps them help you. In response to this, they should return that respect in addressing your issue, assisting with changes, and helping you finalize your pull requests.
 
-Below are the steps needed to add code to BeEF on Github to the master branch:
+###  We want any form helpful of contributions!
 
-1. Fork the repository and create a new branch
 
-2. Write and commit your new code to that branch
+ BeEF is an open source project and we love to receive contributions from the community! There are many ways to contribute, from writing tutorials or blog posts, improving or translating the documentation, answering questions on the project, submitting bug reports and feature requests or writing or reviewing code which can be merged into BeEF itself.
 
-3. Run the existing tests to make sure they pass
+# Contributing
+### Anyone is welcome to make BeEF better!
+Thank you for wanting to contribute to BeEF. It's effort like yours that helps make BeEF such a great tool.
 
-4. Make a pull request from the forked branch to BeEF master
+Following these guidelines shows that you respect the time of the developers developing this open source project and helps them help you. In response to this, they should return that respect in addressing your issue, assisting with changes, and helping you finalize your pull requests.
 
-See https://github.com/beefproject/beef/wiki/BeEF-Testing for steps to write tests.
 
-TL;DR: 
-```
-bundle install --with test
-bundle exec rake spec
-```
+###  We want any form helpful of contributions!
 
-4. Write tests in RSpec for your new code (module, extension etc.)
+ BeEF is an open source project and we love to receive contributions from the community! There are many ways to contribute, from writing tutorials or blog posts, improving or translating the documentation, answering questions on the project, submitting bug reports and feature requests or writing or reviewing code which can be merged into BeEF itself.
 
-5. Run all tests again to make sure they all pass
 
-6. Edit existing wiki page / add a new one explaining the new features, including:
+
+# Ground Rules
+
+###  Responsibilities
+> * When making an issue, ensure the issue template is filled out, failure to do so can and will result in a closed ticket and a delay in support.
+> * We now have a two-week of unresponsiveness period before closing a ticket, if this happens, just comment responding and @ whoever closed the ticket. 
```

Comment:
`... and the issue will be reopened` ?

Reply:
Fixed, changed to be:
" just comment responding to the issue which will re-open the ticket."